### PR TITLE
Secret added to console

### PR DESCRIPTION
**Pull request by muralidhar-chintada**
#24 @suvarna-tammina  I have added the "SECRET" to app.js

### CODE REVIEW

**Review by suvarna-tammina — CHANGES_REQUESTED** [review]
@muralidhar-chintada Add arrow function instead console. look at #23 example and follow that.

**Comment by muralidhar-chintada** [comment]
@suvarna-tammina I have made the changes required. Please check it once.

File: go-ducks.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const consumables = [() => ''];
+const consumables = [() => ' SECRET'];
```
Comment (cwc-products) [inline go-ducks.js]: @muralidhar-chintada It looks like there is a space before SECRET.  Please remove the space.

**Review by cwc-products — DISMISSED** [review]

**Review by suvarna-tammina — APPROVED** [review]
@muralidhar-chintada Looks Good

File: go-ducks.js (hunk 1 above)
Comment (suvarna-tammina) [reply go-ducks.js]: Resolved.

**Review by suvarna-tammina — APPROVED** [review]
@muralidhar-chintada  Resolved

**Pull request merged**